contracts/SportMarkets/SportPositions/SportPositionalMarket.sol

```diff
@@ -339,8 +367,10 @@ contract SportPositionalMarket is OwnedWithInit, ISportPositionalMarket {
 
         _mint(msg.sender, value);
 
-        _incrementDeposited(value);
-        _manager().transferSusdTo(msg.sender, address(this), value);
+        if (!isDoubleChance) {
```

Comment:
It is incorrect to mint all positions for double chance. Rather, AMM should only mint the position it is selling.